app/javascript/packs/email-validation.js

```diff
@@ -5,16 +5,6 @@ function emailValidation() {
   const email = document.querySelector('input[type="email"]');
   let blurTimer;
 
-  function hideElem(elem) {
-    elem.classList.add('hide');
-    elem.hidden = true;
-  }
-
-  function showElem(elem) {
-    elem.classList.remove('hide');
-    elem.hidden = false;
-  }
```

Comment:
I had to add the explicit `hidden` attribute in #3773 because of IE + JAWS accessibility reports, I think we may need to keep it

Reply:
> I had to add the explicit `hidden` attribute in #3773 because of IE + JAWS accessibility reports, I think we may need to keep it

I'll double-check, but if I recall correctly, I discovered that the `hide` class is implemented as something of a screen-reader-only class ([`usa-sr-only` in USWDS](https://designsystem.digital.gov/documentation/accessibility/)), which would explain why it was being read to screen readers.

<img src="https://user-images.githubusercontent.com/1779930/99676758-e56aef80-2a46-11eb-894a-a8cd6d9247b2.png" width="256" alt="hide class">

I revised this to use `display-none` instead, which hides it both visually and from screen readers.

>`display:none` [...] styles will hide content from all users. The content is removed from the visual flow of the page and is ignored by screen readers

https://webaim.org/techniques/css/invisiblecontent/#techniques

Reply:
Yeah maybe the issue was that the `.hide` style didn't use `display: none`, I don't think I dug in that far at the time, thanks for checking! Just wanted to make sure we don't regress and get another issue filed against us

Reply:
> I don't think I dug in that far at the time

To be fair, it's also not the greatest choice of name, since "hide" does seem to imply `display: none`. At least, that was my initial interpretation seeing it for the first time.

Reply:
I was able to test this as working successfully in Safari + VoiceOver. For lack of access to JAWS, I did at least find this reference chart that appears to show that describes the expected behavior, and that JAWS abides by it:

https://stevefaulkner.github.io/HTML5accessibility/tests/hidden-2016.html

Specifically items 3 and 4:

>**4. CSS off screen** content hidden (in supporting browsers) and announced by screen reader

(Current implementation, using `hide` class)

>**3. CSS display:none** content hidden (in supporting browsers) and ignored by screen reader

(Proposed implementation, using `display-none` class)